github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -128,14 +128,13 @@ presubmits:
       affinity:
         podAntiAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Maybe we should use `requiredDuringSchedulingIgnoredDuringExecution` instead, as we do for SR-IOV?

Reply:
Since IIUC preferred rules that are not [satisfiable will be ignored](https://cast.ai/blog/kubernetes-scheduler-how-to-make-it-work-with-inter-pod-affinity-and-anti-affinity/#:~:text=requiredDuringSchedulingIgnoredDuringExecution%20%E2%80%93%20it%20means%20that%20conditions%20must%20be%20satisfied%20to%20pod,be%20ignored%20(soft%20requirement).)?

Reply:
Yes - good call. I will make the update.